### DISCUSSION TITLE
Links to .html files are now requiring the extension?

### DISCUSSION THREAD

**Discussion by devowhippit** [community/community#22999] [Other Feature Feedback, Questions, & Ideas]
<p>Just recently our team noticed some links to static <strong>.html</strong> pages were 404ing on some of our GitHub Pages sites. They were using clean urls without the <strong>.html</strong> extension. For example <strong>/about</strong> for the <strong>/about.html</strong> page. The clean url has been working for the longest time but it seems to have just recently stopped. Was there a change or is this a temporary issue?</p>

**Comment by ThirteenAG** [comment]
<p>Can confirm, needs to be fixed asap.</p>

**Comment by nhatter** [comment]
<p>Duplicate of <a href="https://github.community/t/github-pages-redirecting-all-my-md-pages-to-folders-now/10629">https://github.community/t5/GitHub-Pages/GitHub-Pages-redirecting-all-my-md-pages-to-folders-now/m-p/23904#M1686</a> It appears it’s affecting many GitHub users: <a href="https://twitter.com/theNickHatter/status/1129088073055318018?fbclid=IwAR3kRD_cFqqujqU24wNgwlciUxCXOV2GjgN4hLsRKTdTlAqh3MBUjhCmgqw" rel="nofollow noopener">https://twitter.com/theNickHatter/status/1129088073055318018?fbclid=IwAR3kRD_cFqqujqU24wNgwlciUxCXOV2GjgN4hLsRKTdTlAqh3MBUjhCmgqw</a> TEMP WORKAROUND: - Call /pagename.html directly - Or try putting in your page header: ---- permalink: pagename/ ---- Knew it was only a matter of time before Microsoft screwed up :wink:</p>

**Comment by nhatter** [comment, marked as answer]
<p><strong>UPDATE 16th May 2019 20:16</strong></p>
<p>OK, the redirect problem seems to have stopped.</p>
<p>Tested across Chrome, Firefox and Safari.</p>
<p>Will keep an eye out for an official GitHub response.</p>

**Comment by devowhippit** [comment]
<p>Thanks <a class="mention" href="/nhatter">@nhatter</a>. Accordioning to the other ticket thread it has been fixed.</p>

**Comment by nhatter** [comment]
<p>Yep GitHub fixed it and confirmed there was an issue.</p>
<p><strong>UPDATE 16th May 2019 20:24:</strong></p>
<p>GitHub has confirmed there was an issue and are now deploying a fix:<br>
<a href="https://www.githubstatus.com/incidents/6vyp7d36str6" rel="nofollow noopener">https://www.githubstatus.com/incidents/6vyp7d36str6</a></p>
<p><strong>Official Response from GitHub Support:</strong></p>
<pre><code>Hi Nick,

Thanks for writing in, and we apologize for the trouble this is causing.

This sounds like it may be related to an ongoing status event:

https://www.githubstatus.com

Our team is actively working on fixing it. I'd recommend keeping an eye on our status site so you can check when this is resolved. You can also subscribe to that page in order to be notified when the event is over.

Please let us know if you're still having trouble after our status changes back to green. Note that it may take some additional time after that for things to get caught up.

Thanks,
Kirzi
GitHub Support
</code></pre>

**Comment by nabilateuse** [comment]
<p>GitHub has confirmed there was an issue and are now deploying a fix:<br>
<a href="https://www.githubstatus.com/incidents/6vyp7d36str6" rel="nofollow noopener">https://www.githubstatus.com/incidents/6vyp7d36str6</a></p>
